src/views/consent/ConsentBeacon.js

```diff
@@ -30,20 +30,26 @@ define(['okta'], function (Okta) {
         </div>\
       </div>\
       <div class="logo-wrapper consent-beacon-client">\
+        {{#if clientURI}}\
+          <a href="{{clientURI}}" class="client-logo-link" target="_blank">\
+        {{/if}}\
         <div class="consent-beacon-border"/>\
```

Comment:
rather than try and bring the logo in front of the border div so that it's clickable.. just turn the border div into the link.